src/karts/skidding.cpp

```diff
@@ -325,14 +325,13 @@ void Skidding::update(float dt, bool is_on_ground,
             m_predicted_curve->setVisible(true);
             m_predicted_curve->setPosition(m_kart->getXYZ());
             m_predicted_curve->setHeading(m_kart->getHeading());
-            float angle = kp
-                                ->getMaxSteerAngle(m_kart->getSpeed())
+            //FIXME: why successive assignations with different values ?
```

Comment:
Oh, right I dont intend to let this, @hiker I believe this is you who introduced the code, does it have any side effect outside of this scope ?